src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs

```diff
@@ -34,21 +34,23 @@ public override DeploymentResult Deploy()
                 DotnetPublish();
             }
 
+            var webRootLocation = DeploymentParameters.PublishApplicationBeforeDeployment ? DeploymentParameters.PublishedApplicationRootPath : DeploymentParameters.ApplicationPath;
```

Comment:
I think WebRootLocation was never renamed when we moved web.config from wwwroot to the parent directory. It looks like it should be renamed to AppRoot